simul.spec

```
@@ -48,10 +47,10 @@ and coherence of parallel filesystems.
 %if %{with_mpich}
 %package mpich
 Summary: Simul for MPICH
-%if "%{_host_vendor}" == "suse"
-BuildRequires: mpich-devel%{?_isa} lua-lmod libfabric-devel glibc
+%if (0%{?suse_version} > 0)
+BuildRequires: mpich-devel%{?_isa} lua-lmod libfabric-devel
```

Don't repeat common packages:

```suggestion
BuildRequires: lua-lmod libfabric-devel
```